### DISCUSSION TITLE
*Loading... Connecting to console...* Mac OS Ventura 13.1

### DISCUSSION THREAD

**Discussion by ToxicCallum** [unknownskl/xbox-xcloud-client#643] [General]
_Currently Running 2.0.0-beta2 pre-release_

This application used to run like a dream on the 21" Mac I am using.

But ever since _December 7th 2022,_ I can no longer connect to my console.

I am currently trying out many ways to fix this.

Here is a list of failed methods so far:

- Uninstall, reinstall
- Ensuring Xbox Live membership is current
- Login & logging out
- Editing the bit rate
- Reconnecting to the internet on both **Xbox Series X** & **Mac**

At this current stage, I am not sure if the new Mac OS is the problem, or if the Application is bugged & I need to do something different.

If anybody can help, or having the same sort of issues, please comment so we can figure this out as the Application is the best currently available for this on Mac.

**Comment by ToxicCallum** [comment]
**Remote play on iOS devices (Known Issue)**

_Hey Gamers,_


_There is currently a known issue when using remote play to connect to a console over cellular or external Wi-Fi networks on devices running iOS 16 and above. We understand this is a frustrating issue for players who enjoy remote play on the go._


_As this is currently affecting users over cellular or external Wi-Fi networks via remote play, players with an active Game Pass Ultimate subscription can utilize cloud gaming by going to https://xbox.com/play to continue mobile gaming._


_There is currently no ETA for a solution. We will continue to share updates here as we make progress._


_Thanks for your patience._


https://answers.microsoft.com/en-us/xbox/forum/all/remote-play-on-ios-devices-known-issue/042131a3-d7c0-4df9-9068-5864ea3e89f2

**Reply by rackerby** [reply]
There was an Xbox update released on December 6, 2022.

https://support.xbox.com/en-US/help/hardware-network/settings-updates/whats-new-xbox-one-system-updates

That said, 2.0.0-beta2 still allows me to connect to my Series X from a Mac running 12.6.2 (Monterey) as long as I'm on the same LAN.

**Comment by ToxicCallum** [comment]
**UPDATE**

As of today (Wednesday 28th December) (GMT+12)

The application is working perfectly again!


Will continue to use this thread to update on any known issues I'm experiencing.
